data/supergroups.yml

```diff
@@ -0,0 +1,27 @@
+content_purpose_supergroup:
+  name: "Content Purpose Supergroup"
+  description: "These are the new super groups that came out of the research from the JTBD framework in Q4 by the navigation team and are used in the topic pages."
```

Comment:
Need to remove the space between `super groups` to be `supergroups`

Reply:
Cheers! 👀